src/constants/examples.constant.ts

```diff
@@ -18,4 +18,5 @@ export const EXAMPLES = {
   YEAR_BASIS: '1',
   INDEX_RATE_CHANGE_FREQUENCY: '1',
   LOAN_BILLING_FREQUENCY_TYPE: 'C',
+  TARGET_AMOUNT: 5040,
```

Comment:
Can we reuse existing example constant DEAL_OR_FACILITY_VALUE.

ACBS has a lot of different names for quite similar information. Facility and Covenant amount/value should match in most cases.

Reply:
Removed `TARGET_AMOUNT` constant and reused `DEAL_OR_FACILITY_VALUE` instead